Define neo colonialism (#360)
tatianamac/selfdefined: 11ty/definitions/neo-colonialism.md

---
title: neo-colonialism
slug: neo-colonialism
speech: noun
excerpt: the continued influence of colonial era ideologies and practices in a country or region by former dependencies, generally enforced by its own government and/or citizens as a lingering effect of colonialism

> sub_terms:
>     - text: neo-
>     full_title: neo-colonialism
>     - text: modern
>     full_title: modern colonialism

Reviewer: ```suggestion
skip_in_table_of_content: true
```
Because this term is itself a sub-term, we will use this line of front-matter in order to make it skipped in the table of contents.

defined: true
reading:

>     - text: neo-colonialim - internet encyclopedia of philosophy
>       href: https://iep.utm.edu/neocolon/

Reviewer: ```suggestion
  - text: neo-colonialim - internet encyclopedia of philosophy
    href: https://iep.utm.edu/neocolon/
```
The tabbing is a bit particular in front matter 🙃 Making these updates will make your repo build.

Author: got it - will make a note of this for the future

---

> the continued influence of colonial era ideologies and practices in a country or region by former dependencies, generally enforced by its own government and/or citizens as a lingering effect of colonialism

Reviewer: ```suggestion
the continued influence and impact of colonial-era ideologies and practices in a country or region by former dependencies, generally enforced by its own government and/or citizens as a lingering effect of [colonialism](/definitions/colonialism)
```
Adds cross-linking with colonialism, which will be live once we merge this in!

### Note

> An extension of colonialism that appears in the 21st century is neo-colonialism - a more pervasive form of imposing ideological and economic dominance on a country after it has been declared independent from its colonizer. The stealth nature of neo-colonialism can be traced to treaties, agreements, and ideologies that have been imparted from the oppresors on the oppressed because of previous dependencies.

Reviewer: ```suggestion
Neo-colonialism is a modern and more pervasive form of imposing ideological and economic dominance on a country after it has been declared independent from its coloniser. The stealth nature of neo-colonialism can be traced to treaties, agreements, and ideologies that have been imparted from the oppressors onto the oppressed from previous dependencies.
```

> For instance, in most African nations that were colonized by the British empire, western countries' ideals are seen as civilized and the best representation of societal progress. This erases and dismisses other ways of life, deeming them unconventional. Another form of neo-colonial influence can be seen in how governments limit the freedom of speech and other civil liberties of their citizens. A recent manifestation of this is seen in the [2020 End SARS campaign in Nigeria](https://en.wikipedia.org/wiki/End_SARS).

Reviewer: ```suggestion
### Impact
In most African nations that were colonised by the British empire, western countries' ideals are seen as civilized and the best representation of societal progress. Standards of living that are ancestral are often dismissed as uncivilised and unconventional, erasing what is often thousands of years of culture.

Governments limiting freedom of speech and other civil liberties of their citizens. A recent manifestation of this is seen in the [2020 #EndSARS campaign in Nigeria](https://en.wikipedia.org/wiki/End_SARS).
```
